Android/CoilBase/cgmanifest.json

```diff
@@ -0,0 +1,16 @@
+{
+  "Registrations": [
+    {
+      "Component": {
+        "Type": "Maven",
+        "Maven": {
+          "ArtifactId": "coil-base",
+          "GroupId": "io.coil-kt",
+          "Version": "1.3.2",
+          "NuGetId": "Xamarin.Io.Coil.Kt.Coil.Base"
```

Comment:
@moljac I think this field should also be updated to `Xamarin.Io.CoilKt.CoilBase`, or not?

Reply:
yeah. thanks!

not critical for publishing nuget. will do it on main branch directly.

Reply:
Nice.
To make it easier for you I updated https://github.com/xamarin/XamarinComponents/pull/1251 with all the fixes you made in this branch .